src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -213,6 +213,9 @@
         - name: --ppg
           type: string
           short-summary: The ID of a PPG.
+        - name: --enable-fips-image
```

Comment:
Forgot to ask, does this parameter --enable-fips-image only appear in 'az aks create...' and 'az aks nodepool add...', but not appear in upgrade/scale/update commands?